blueprints/node-webkit/index.js

```diff
@@ -45,7 +45,7 @@ module.exports = {
       var json = JSON.parse(data);
       json.main = 'dist/index.html';
       json.window = { width: 960, height: 600 };
-
+      json.devDependencies['socket.io-client'] = '^1.3.5';
```

Comment:
We should be able to remove this dependency here too, given we can use the one that Testem provides.